README.md

```diff
@@ -1,151 +1,149 @@
-# Coq platform
+# Coq Platform 2021.09.0
 
-The Coq platform is a distribution of the Coq proof assistant together with a selection of Coq libraries.
-It provides a set of scripts to compile and install OPAM, Coq, Coq libraries and Coq plugins on MacOS,
+The [Coq proof assistant](https://coq.inria.fr) provides a formal language
+to write mathematical definitions, executable algorithms, and theorems, together
+with an environment for semi-interactive development of machine-checked proofs.
+
+The **Coq Platform** is a distribution of the Coq proof assistant together
+with a selection of Coq libraries and plugins.
+The main goal of the Coq Platform is to provide a system independent, dependable
+and easy to install platform for developments based on the Coq proof assistant
+and for teaching Coq.
+See [Charter](charter.md) for the Coq Platform concept.
+
+The Coq Platform is based on the OCaml package manager **opam** and provides a set
+of scripts to compile and install OPAM, Coq and the platform contents on macOS,
 Windows and many Linux distributions in a reliable way with consistent results.
+In addition **pre-compiled binary packages** or **installers** are provided for **macOS**,
+**Windows** and **snap** for Linux (Docker is in preparation).
 
-See [Charter](charter.md) for the coq platform concept.
+The Coq Platform supports to install several versions of Coq - also in parallel,
+e.g. for porting developments from one version of Coq to another.
+The table below contains links to the README files for the supported versions
+of Coq and libraries. Each README file contains a list of included packages with
+detailed information for each package.
 
-# Note on Licenses
+- [Coq 8.13.2 04/2021 with updated/extended package pick 09/2021](doc/README_2021.09.0_8.13.md)
+- [Coq 8.13.2 04/2021 with original package pick 02/2021](doc/README_2021.09.0_8.13~2021.02.md)
+- [Coq 8.12.2 12/2020 (previous release version)](doc/README_2021.09.0_8.12.md)
```

Comment:
I suggest being consistent in the way we refer to platform versions (and to use a different style when talking about other dates, e.g., release dates of Coq). Something like the following:

```suggestion
- [Coq 8.13.2 (released Apr 2021) with updated/extended package pick 2021.09](doc/README_2021.09.0_8.13.md)
- [Coq 8.13.2 (released Apr 2021) with original package pick 2021.02](doc/README_2021.09.0_8.13~2021.02.md)
- [Coq 8.12.2 (released Dec 2020)](doc/README_2021.09.0_8.12.md)
```